ibis/expr/operations/relations.py

```diff
@@ -495,7 +494,7 @@ def _projection(self):
 @public
 class DummyTable(Relation):
     # TODO(kszucs): verify that it has at least one element: Length(at_least=1)
-    values: VarTuple[Value[dt.Any, ds.Scalar]]
+    values: VarTuple[Value[dt.Any]]
```

Comment:
Why is this change required?

BTW the original could be spelled as `VarTuple[Scalar]`

Reply:
Otherwise it's not possible to write `ibis.range(n).unnest()`. I need to be able to create a table from an unnested array without existing base relations, and `unnest(x)` is `ds.Columnar`.

Reply:
Why not converting to table *before* calling unnest?

```py
In [9]: ibis.range(5).as_table()[0].unnest()
Out[9]:
r0 := DummyTable
  IntegerRange(0, 5, 1) array<int8>

IntegerRange(0, 5, 1): Unnest(r0.IntegerRange(0, 5, 1))
```

Reply:
That UX is not really acceptable though 😂

Reply:
It's perfectly valid to have column expressions that have no base relation, though it's not particularly common.